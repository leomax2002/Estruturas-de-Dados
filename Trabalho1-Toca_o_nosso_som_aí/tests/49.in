add FourSeasons
add OdetoJoy
add RideoftheValkyries
add ToccataeFugue
add CanonemRe
add NessunDorma
play
add OFortuna
ended
stop
del RideoftheValkyries
undo *
del OFortuna
stop
del OdetoJoy
current
fight