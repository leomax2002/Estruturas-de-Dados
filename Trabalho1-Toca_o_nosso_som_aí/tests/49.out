RideoftheValkyries
Jedi Wagner, assuma o comando!